product-service/handlers/getProductsById.ts

```diff
@@ -0,0 +1,34 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import 'source-map-support/register';
+import productList from '../mocks/productList.mock.data.json';
+const headers = {
+    // "Access-Control-Allow-Origin": "https://d3kaeffpjwwpyk.cloudfront.net"
+    "Access-Control-Allow-Origin": "*"
+};
+
+export const getProductsById: APIGatewayProxyHandler = async (event, _context) => {
+    try {
+
+        const req_id = event.pathParameters.productId;
+        const product = productList.find(({ id: db_id }) => db_id === req_id);
+        if (product) {
+            return {
+                headers,
+                statusCode: 200,
+                body: JSON.stringify(product)
+            }
+        } else {
+            return {
+                headers,
+                statusCode: 200,
```

Comment:
Good point. I put that 200 because of expected that it is better. But I am not so experimenting in development. Nevermind, I will change it into 404. Thanks.